<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>622</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1053</width>
    <height>741</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>557</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="maximumSize">
       <size>
        <width>782</width>
        <height>539</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>파일(F)</string>
    </property>
    <addaction name="actionn"/>
    <addaction name="action_W"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="actione_4"/>
    <addaction name="separator"/>
    <addaction name="actione_5"/>
    <addaction name="actione_6"/>
    <addaction name="separator"/>
    <addaction name="actione_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>편집(E)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>서식(O)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>보기(V)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>확대하기/축소하기</string>
     </property>
     <addaction name="actionASD"/>
     <addaction name="actioncnrth"/>
     <addaction name="actionghkrreogkrl_cntrh"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="actionzxc"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>도움말(H)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>기</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>ㅁ</string>
   </property>
  </action>
  <action name="actionzxc">
   <property name="text">
    <string>상태표시줄(S)</string>
   </property>
  </action>
  <action name="actionASD">
   <property name="text">
    <string>확대(I)</string>
   </property>
  </action>
  <action name="actioncnrth">
   <property name="text">
    <string>축소(O)</string>
   </property>
  </action>
  <action name="actionghkrreogkrl_cntrh">
   <property name="text">
    <string>확대하기/축소하기 기본값 복원</string>
   </property>
  </action>
  <action name="actionn">
   <property name="text">
    <string>새로 만들기(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>새 창(W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>열기(O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>저장(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actione_4">
   <property name="text">
    <string>다른 이름으로 저장(A)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actione_5">
   <property name="text">
    <string>페이지 설정(U)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actione_6">
   <property name="text">
    <string>인쇄(P)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actione_X">
   <property name="text">
    <string>끝내기(X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_W</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
